app/models/destination.rb

```diff
@@ -25,6 +27,19 @@ def current_destination
     service.flow_object(flow_uuid).default_next
   end
 
+  def both_pages_present?
```

Comment:
on second thoughts this should be a little more specific. `cya_and_confirmation_present?` perhaps?